inf-clojure.el

```diff
@@ -940,6 +940,25 @@ prefix argument PROMPT-FOR-SYMBOL, it prompts for a symbol name."
                (inf-clojure-symbol-at-point))))
     (comint-proc-query (inf-clojure-proc) (format (inf-clojure-var-source-form) var))))
 
+(defvar inf-clojure--dump-file-name ".inf-clojure.dump"
```

Comment:
Well I am not sure it is a debug mode...just dumping stuff. Anyways it is ok to me to call it debug.

About the file, I like that I can `tail -f` but I guess there is a way to save to disk or to tail once in a buffer without too much effort.

The tail is fundamental and saved me a couple of times already. I have had this feature for long time locally now.